_restore_db.sh

```diff
@@ -8,4 +8,4 @@
 psql -d dengue < alertademo_data.sql
 
 #gzip -d infodengue.sql.gz
-#psql -d infodengue < infodengue.sql
\ No newline at end of file
+#psql -d infodengue < patch\infodengue.sql
```

Comment:
essa linha nao deveria ser alterada é um comentário referente ao restore da base infodengue.
adicione na linha 9 o comando que deveria ser algo como `psql -d < patch/notificacao.sql`

Reply:
Sorry, it's already corrected.
Thanks!